miner/worker.go

```diff
@@ -1459,3 +1482,52 @@ func signalToErr(signal int32) error {
 		panic(fmt.Errorf("undefined signal %d", signal))
 	}
 }
+
+// generateDAGTx generates a DAG transaction for the block
+func (w *worker) generateDAGTx(env *environment) (*types.Transaction, error) {
+	// @TODO this is a placeholder for the txDAG transaction sender
+	privateKeyHex := "05305d9000ae59abb613799237e87885dd122bafd71beffb60cc15ebf95d1d1f" //0x0fC7B89DdE80018CaF9eEd78757f996d9d3a25Aa
```

Comment:
Do from and to have any special meaning?

Reply:
No，just two normal addresses.